docs/controls/InAppNotification.md

```diff
@@ -0,0 +1,129 @@
+# In App Notification 
+
+The *In App Notification* control offers the ability to show local notifications in your application.
+
+## Syntax
+
+The control should be placed where you want your notification to be displayed in the page, generally in the root grid.
```

Comment:
We can always add a note in the docs if that will make things easier. How about:

>Note: Since the control is part of the page visual tree, it will render in the order it was added in the parent control, and might be hidden by other elements. For the control to render on top of other elements, add it as the last child of the parent control or set the Canvas.ZIndex to a high number.